zebra/zebra_pbr.h

```diff
@@ -171,6 +171,9 @@ struct zebra_pbr_iptable {
 
 	struct list *interface_name_list;
 
+#define IPTABLE_INSTALL_IN_PROGRESS 1 << 1
```

Comment:
small nit: Could we call this IPTABLE_INSTALL_QUEUED to keep it consistent with other dplane object install state flags like routes?

Reply:
done. thanks for review.